php_memcached.stub.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/**
+ * @generate-function-entries
+ * @generate-legacy-arginfo
+ */
+
+
+class Memcached {
+
+	public function __construct(string $persistent_id=NULL, callable $callback=NULL, string $connection_str=NULL) {}
+
+	public function getResultCode(): int {}
+	public function getResultMessage(): string {}
+
+	public function get(string $key, callable $cache_cb=NULL, int $get_flags=0): mixed {}
+	public function getByKey(string $server_key, string $key, callable $cache_cb=NULL, int $get_flags=0): mixed {}
+	public function getMulti(array $keys, int $get_flags=0): false|array {}
```

Comment:
I've just accidentally come across this PR, and noticed the `MEMC_METHOD_FETCH_OBJECT` macro (mostly because PHP 8 throws an `Error` for most uninitialized objects, e.g.: https://github.com/php/php-src/blob/74fe9170b65740bcdc41c9706ec38c31654c12f6/ext/pdo/pdo_stmt.c#L37).

So the reason why I'm writing is that it seems to me that the return in `MEMC_METHOD_FETCH_OBJECT` is not reflected in the return types in the stubs. Unless I'm missing something, the macro can result in a `null` return value.

Reply:
Your are right and I think memcached should also raise an exception in such case (and thus, don't pollute arginfo with null case)

Reply:
@kocsismate see pr #465

Reply:
@remicollet Looks good, I agree that it's the best thing to do :)